### ISSUE TITLE
Use webgl-debug in debug mode

### ISSUE THREAD

**Issue by vorg** [pex-gl/pex-context#40]
Using https://www.npmjs.com/package/webgl-debug would allow use to catch invalid enums and other gl errors.

**Comment by dmnsgn** [comment]
Is there any advantage over the current `checkError()`?

**Comment by vorg** [comment]
Yes it checks for invalid enums and parameters of function calls. So you can catch error when it happens not at some point in the future or end of frame.
